contracts/yield-source/YearnV2YieldSource.sol

```diff
@@ -261,7 +313,7 @@ contract YearnV2YieldSource is IYieldSource, ERC20Upgradeable, OwnableUpgradeabl
     /// @dev used to calculate how many shares to mint / burn when depositing / withdrawing
     /// @return tokens number of tokens equivalent (in value) to the amount of Yield Source shares
     function _sharesToToken(uint256 shares) internal view returns (uint256 tokens) {
-        if(totalSupply() == 0) {
+        if (totalSupply() == 0) {
```

Comment:
let's pull this into a variable, as it's a public function call

Reply:
Done: https://github.com/pooltogether/pooltogether-yearnv2-yield-source/pull/9/commits/32b130dbbaf28b614075ce2d30b0ee7c9f47593c